infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/LoggingConfig.java

```diff
@@ -73,17 +79,51 @@ public String getLogFileNamePattern() {
   }
 
   public static final class LoggingConfigBuilder {
+    public static final String SEP = System.getProperty("file.separator");
+
+    private boolean colorEnabled = true;
+    private boolean includeEventsEnabled = true;
+    private boolean includeValidatorDutiesEnabled = true;
+    private boolean includeP2pWarningsEnabled = false;
+    private LoggingDestination destination = LoggingDestination.DEFAULT_BOTH;
 
-    private boolean colorEnabled;
-    private boolean includeEventsEnabled;
-    private boolean includeValidatorDutiesEnabled;
-    private boolean includeP2pWarningsEnabled;
-    private LoggingDestination destination;
-    private String logFile;
+    private String logFileNamePrefix = DEFAULT_LOG_FILE_NAME_PREFIX;
+    private String logFileName;
     private String logFileNamePattern;
 
+    private String dataDirectory;
+    private String logDirectory;
+    private String logPath;
+    private String logPathPattern;
+
     private LoggingConfigBuilder() {}
 
+    private void initMissingDefaults() {
+      if (logPath == null || logPathPattern == null) {
+        if (logFileName == null) {
+          logFileName = logFileNamePrefix + DEFAULT_LOG_FILE_NAME_SUFFIX;
+        }
+        if (logFileNamePattern == null) {
+          logFileNamePattern = logFileNamePrefix + DEFAULT_LOG_FILE_NAME_PATTERN_SUFFIX;
+        }
+
+        if (logDirectory == null) {
+          if (dataDirectory == null) {
+            logDirectory = DEFAULT_LOG_DIRECTORY;
```

Comment:
I'm not a huge fan of defaulting to the current directory for logs if nothing is defined. It's almost certainly going to be the wrong place and often result in logs not being written at all due to permissions issues.  It think throwing an exception would be a better option in this case so you either have to specify a `logDirectory` or specify `dataDirectory`.

Reply:
Yep, makes sense to me. Changed here: a639e30